frontend/.eslintrc.cjs

```diff
@@ -20,8 +20,8 @@ module.exports = {
       // React sources
       files: ['**/*.{js,jsx,ts,tsx}'],
       excludedFiles: ['./tmp/**'],
-      plugins: ['react', 'jsx-a11y'],
-      extends: ['plugin:react/recommended', 'plugin:react/jsx-runtime', 'plugin:react-hooks/recommended', 'plugin:jsx-a11y/recommended', 'prettier'],
+      plugins: ['react'],
+      extends: ['plugin:react/recommended', 'plugin:react/jsx-runtime', 'plugin:react-hooks/recommended', 'prettier'],
```

Comment:
These eslint plugins aren't using axe.. they should be left in the project so we get some warnings about a11y issues in the .tsx files.